js/controllers/hotkeys.js

```diff
@@ -46,7 +49,7 @@ HotkeysController.prototype.onKeydown_ = function(e) {
         return false;
 
       case this.KEY.F:
-        $('#search-button').click();
+        document.getElementById('search-input').click();
```

Comment:
Part of me wonders whether you should use `focus()` instead.

Reply:
Good call. I'll change the listener to look for focus too